Sources/NiceComponents/Components/LoadingView.swift

```diff
@@ -8,23 +8,28 @@
 import SwiftUI
 import UIKit
 
-public struct LoadingView: View {
-    var loadingText: String? = "Loading"
+public struct LoadingView<Footer: View>: View {
+    let style: UIActivityIndicatorView.Style
+    let footer: () -> Footer
+
     @State var playAnimation = true
```

Comment:
This isn't being used, let's either remove it or turn it into a binding to control the animation